packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -586,6 +587,7 @@ class _Decoration {
   final _InputBorderGap borderGap;
   final bool alignLabelWithHint;
   final bool? isDense;
+  final bool? isEmpty;
```

Comment:
Could we make this non-nullable?

```suggestion
  final bool isEmpty;
```

Reply:
Yes, good catch!